vitest.setup.ts

```diff
@@ -0,0 +1,4 @@
+export const setup = () => {
+    // UTC−09:00
+    process.env.TZ = 'Etc/GMT+9';
```

Comment:
I think using Eastern Standard Time might make more sense for tests considering the project's eastern Canadian roots? 🍁